mlflow/pytorch/__init__.py

```diff
@@ -268,6 +298,27 @@ def save_model(
                           serialized to json using the Pandas split-oriented format. Bytes are
                           base64-encoded.
 
+    :param requirements_file: A string containing the path to requirements file. Remote URIs
```

Comment:
Can we store `requirements_file` and `extra_files` in the `MLmodel` file so we can look them up later?

Reply:
@harupy Adding to MLmodel has performance implications during model inference loading so these files need to be kept as separate artifacts, but bundled along with the model for version tracking and matching artifacts

Reply:
@charnger 

> these files need to be kept as separate artifacts

Yes, they will be kept as separate files and `MLmodel` file (which contains metadata of a logged model) should contains information of where they are located like below.

```yml
# Example of MLmodel file
artifact_path: models
flavors:
  python_function:
    data: data
    env: conda.yaml
    loader_module: mlflow.pytorch
    pickle_module_name: mlflow.pytorch.pickle_module
    python_version: 3.6.10
  pytorch:
    model_data: data
    pytorch_version: 1.4.0
    # new fields
    requirements_file: ...
    extra_files:
      - ...
      - ...
run_id: cd8c2872466c463790bc4ac70b9d161f
utc_time_created: '2020-10-27 23:22:47.135800'

```